src/components/context/UserContext.jsx

```diff
@@ -0,0 +1,30 @@
+import fetchUser from '../../services/user'
+
+const { createContext, useState, useEffect, useContext } = require('react')
```

Comment:
It appears to be working but I would be consistent and use the `import` syntax rather than `require`